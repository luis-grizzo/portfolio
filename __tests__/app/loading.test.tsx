import { render } from '@testing-library/react'

import Loading from '@/app/loading'

describe('Loading', () => {
  it('should match snapshot', () => {
    const { container } = render(<Loading />)

    expect(container).toMatchSnapshot()
  })
})
