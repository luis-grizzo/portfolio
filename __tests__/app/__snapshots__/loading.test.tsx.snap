// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match snapshot 1`] = `
<div>
  <main
    class="flex flex-col items-center justify-center gap-8 md:gap-8 lg:gap-16 h-[calc(100vh_-_114px_-_88px_-_1px)] container mx-auto mt-[114px] py-8 md:py-16 lg:py-32 px-4"
  >
    <img
      alt=""
      class="w-16 aspect-square"
      data-nimg="1"
      decoding="async"
      height="40"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent;"
      width="40"
    />
    <svg
      class="animate-spin"
      fill="currentColor"
      height="32"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 1024 1024"
      width="32"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 0 0-94.3-139.9 437.71 437.71 0 0 0-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
      />
    </svg>
  </main>
</div>
`;
